src/Integration.UnitTests/MefServices/ActiveSolutionBoundTrackerTests.cs

```diff
@@ -544,6 +603,29 @@ public void SolutionBindingUpdated_WhenSetBoundProject_EventsRaisedInExpectedOrd
                 eventCounter.RaisedEventNames[1].Should().Be("SolutionBindingUpdated");
             }
         }
+        
+        [TestMethod]
+        public void HandleBindingChange_WhenSetBoundProject_EventsRaisedInExpectedOrder()
+        {
+            // Arrange
+            using (var testSubject = CreateTestSubject(this.host, this.activeSolutionTracker, this.configProvider, loggerMock.Object))
+            {
+                var eventCounter = new EventCounter(testSubject);
+
+                // Act
+                testSubject.HandleBindingChange(false);
+                
+                // Assert
+                eventCounter.PreSolutionBindingUpdatedCount.Should().Be(1);
+                eventCounter.SolutionBindingUpdatedCount.Should().Be(1);
+                eventCounter.PreSolutionBindingChangedCount.Should().Be(0);
+                eventCounter.SolutionBindingChangedCount.Should().Be(0);
+
+                eventCounter.RaisedEventNames.Should().HaveCount(2);
+                eventCounter.RaisedEventNames[0].Should().Be("PreSolutionBindingUpdated");
```

Comment:
Use name of instead of hardcoded string for event name.
Both in here and in the line 626